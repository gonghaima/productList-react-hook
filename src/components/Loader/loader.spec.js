import React from "react";
import { shallow } from "enzyme";
import { Loader } from "./index";

it("Loader component renders without crashing", () => {
  const ld = shallow(<Loader />);
  expect(ld.debug()).toMatchSnapshot();
});
