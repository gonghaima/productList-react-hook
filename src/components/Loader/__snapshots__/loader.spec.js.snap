// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component renders without crashing 1`] = `
"<div className=\\"Loader\\">
  <img src=\\"logo.svg\\" alt=\\"spinner\\" />
  <p>
    Loading ...
  </p>
</div>"
`;
